### PR TITLE
Fixes URL for organizations

### PR DESCRIPTION
**Pull request by javierarce**
This PR appends the `/u/username` to organizations users to fix a problem requesting images for private visualizations.

### CODE REVIEW

**Comment by javierarce** [comment]
@CartoDB/frontend: review this, please please.

File: lib/assets/javascripts/cartodb/new_dashboard/mapcard_preview.js [hunk 1]
```diff
@@ -31,6 +32,10 @@ module.exports = cdb.core.View.extend({
       url = url + "?api_key=" + this.api_key;
     }
 
+    if (this.options.user && this.options.user.isInsideOrg()) {
+      url = "/u/" + this.options.user.get("username") + url;
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/mapcard_preview.js]: this.user should be ok :)

File: lib/assets/javascripts/cartodb/table/static_image_dialog.js [hunk 2]
```diff
@@ -91,6 +91,10 @@ cdb.admin.StaticImageDialog = cdb.admin.BaseDialog.extend({
       this.url = this.url + "?api_key=" + this.api_key;
     }
 
+    if (this.options.user && this.options.user.isInsideOrg()) {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/static_image_dialog.js]: Not sure if you have ```this.user``` set as you have in the previous one.

File: lib/assets/javascripts/cartodb/new_dashboard/mapcard_preview.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/new_dashboard/mapcard_preview.js]: Ups, you're right. I'll change it, thanks!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1991/))

**Comment by xavijam** [comment]
:)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1993/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1995/))

**Pull request merged**